pdfrw/pdfwriter.py

```diff
@@ -259,8 +259,6 @@ def addpage(self, page):
             obj = obj.Parent
         return self
 
-    addPage = addpage  # for compatibility with pyPdf
```

Comment:
I like to keep a very small amount of pyPdf compatibility for my own testing purposes.